pepper-apis/dsm-core/src/main/resources/liquibase/rgp/PEPPER-833_rgp_kit_display_name.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="PEPPER-833_rgp_kit_type_display_name" author="pegah">
+        <addColumn tableName="kit_type">
+            <column name="display_name" type="VARCHAR(500)"></column>
+        </addColumn>
+    </changeSet>
+    <changeSet id="PEPPER-833_rgp_blood" author="pegah">
+        <insert tableName="kit_type">
+            <column name="kit_type_name" value="BLOOD"></column>
+            <column name="bsp_material_type" value="Whole Blood Streck Cell-Free Preserved"></column>
+            <column name="bsp_receptacle_type" value="Vacutainer Cell-Free DNA Tube Camo-Top [10mL]"></column>
+            <column name="requires_insert_in_kit_tracking" value="1"></column>
+            <column name="display_name" value="BLOOD &amp; RNA"></column>
```

Comment:
Probably safer to replace `&` with `and`.  Might not make any difference, but might save us from some obscure character encoding/escaping headaches.

Reply:
Agreed, do you think GP is going to be ok with it? (I'm not sure where we are in terms of if we have shared screen shots with them for their sop )

Reply:
Good question--can you ask them directly?